lib/opener.ts

```diff
@@ -0,0 +1,13 @@
+///<reference path=".d.ts"/>
```

Comment:
Done with https://github.com/telerik/mobile-cli-lib/commit/3680ec1545a8d02633963c2ca221f158ffeca183 and https://github.com/NativeScript/nativescript-cli/commit/aff7ef3e7a1b6276598e12062177dca8e21e340b